components/ui/FlashCards/MultipleChoice.tsx

```diff
@@ -0,0 +1,215 @@
+import { Text, View, StyleSheet, Pressable } from 'react-native';
+import React, { useState, useMemo } from 'react';
+import { Link } from 'expo-router';
+import { faker } from '@faker-js/faker';
+import MainButton from '../Buttons/MainButton';
+import FlipCard from 'react-native-flip-card';
+
+const MultipleChoice = () => {
+  const [isFlipped, setIsFlipped] = useState(false);
+  const [selectAnswer, setSelectAnswer] = useState('');
+  const [backgroundColorA, setBackgroundColorA] = useState('');
+  const [backgroundColorB, setBackgroundColorB] = useState('');
+  const [backgroundColorC, setBackgroundColorC] = useState('');
+  const [backgroundColorD, setBackgroundColorD] = useState('');
+  const [nextButtonPressed, setNextButtonPressed] = useState(false);
```

Comment:
Instead of multiple `useState` for each background color, would we use an object like:
`
const initialColorState = {
  A: "",
  B: "",
  C: "",
  D: ""
}

const [backgroundColor, setBackgroundColor] = useState<typeof initialColorState>(initialColorState);

`

Reply:
Totally agree, once I got the first one to work I went all copy pasta